clients/src/main/java/org/apache/kafka/clients/consumer/ConsumerConfig.java

```diff
@@ -222,6 +223,17 @@
     private static final String EXCLUDE_INTERNAL_TOPICS_DOC = "Whether records from internal topics (such as offsets) should be exposed to the consumer. "
                                                             + "If set to <code>true</code> the only way to receive records from an internal topic is subscribing to it.";
     public static final boolean DEFAULT_EXCLUDE_INTERNAL_TOPICS = true;
+
+    /** <code>isolation.level</code> */
+    public static final String ISOLATION_LEVEL_CONFIG = "isolation.level";
+    public static final String ISOLATION_LEVEL_DOC = "<p>Controls how to read messages written transactionally. If set to <code>READ_COMMITTED</code>, the application will only receive" +
+            " transactional messages which have been committed. If set to <code>READ_UNCOMMITTED</code>' (the default), the application will receive all messages, even transactional messages" +
+            " which have been aborted. Non-transactional messages will be returned as they are received in either mode.</p> <p>Messages will be  always returned in offset order. Hence, in " +
```

Comment:
nit: `will be returned as they are received` -> maybe just `will be returned in either mode`?